design-editor/src/system/stage-manager.js

```diff
@@ -17,9 +17,10 @@ import {PreviewToolbarElement} from '../panel/preview/preview-toolbar-element';
 // import {InteractionViewToolbarElement} from '../panel/preview/interaction-view-toolbar-element';
 import {InfoElement} from '../pane/select-layer/info-element';
 
-const CompositeDisposable = editor.CompositeDisposable;
-const DESIGN = ViewType.Design;
-const vscode = window.vscode;
+const CompositeDisposable = editor.CompositeDisposable,
+	DESIGN = ViewType.Design,
+	brackets = window.brackets || window.top.brackets,
+	PreferencesManager = brackets ? brackets.getModule('preferences/PreferencesManager') : {};
```

Comment:
I think we don't need these conditions on brackets.getModule().
AFAIK we mocked brackets API and brackets will always return an object.